components/sendgrid/actions/get-all-bounces/get-all-bounces.js

```diff
@@ -0,0 +1,29 @@
+const sendgrid = require("../../sendgrid.app");
+
+module.exports = {
+  key: "sendgrid-get-all-bounces",
+  name: "Get All Bounces",
+  description: "Allows you to get all of your bounces.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    sendgrid,
+    startTime: {
+      type: "integer",
+      label: "Start Time",
+      description:
+        "Refers start of the time range in unix timestamp when a bounce was created (inclusive).",
+      optional: true,
+    },
+    endTime: {
+      type: "integer",
+      label: "End Time",
+      description:
+        "Refers end of the time range in unix timestamp when a bounce was created (inclusive).",
+      optional: true,
+    },
```

Comment:
Why are we not validating these? Are we relying on the Sendgrid API call to do it for us?
Perhaps we could do some basic validation at least, for example make sure that:
- Both numbers are greater than 0
- `startTime` is greater than `endTime`

Reply:
Ok, adding some basic validation.